forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt

```diff
@@ -8,34 +8,27 @@
 package org.forgerock.android.auth
 
 import android.content.Context
-import android.content.SharedPreferences
-
-/**
- * Listener to load the options from persistence for the given context
- */
-interface ConfigInterface {
-    fun loadFromPreference(context: Context): FROptions
-}
 
 /**
  * ConfigHelper Provide the helper methods to persist and load the configurations on start of the SDK
  * This class also verify if the configuration changes on the current and previous launch.
  */
-internal class ConfigHelper: ConfigInterface {
+internal class ConfigHelper {
 
     companion object {
 
-         const val realm = "realm"
-         const val url = "url"
-         const val cookieName = "cookieName"
-         const val clientId = "client_id"
-         const val revokeEndpoint = "revoke_endpoint"
-         const val endSessionEndpoint = "end_session_endpoint"
-         const val scope = "scope"
-         const val redirectUri = "redirect_uri"
+        private  const val realm = "realm"
```

Comment:
super minor, the `private  const val realm = "realm"` has two spaces before const

Reply:
done